library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionActivity.kt

```diff
@@ -149,14 +150,11 @@ internal class TransactionActivity : BaseChuckerActivity() {
     }
 
     companion object {
+        private const val EXTRA_TRANSACTION_ID = "transaction_id"
 
-        private const val ARG_TRANSACTION_ID = "transaction_id"
-        private var selectedTabPosition = 0
```

Comment:
Oh! I just found something... interesting 👀
I guess the previous developer used `selectedTabPosition` as a `static` in order to `store` in memory. And used it when view needs to re-open (in `setupViewPager`).

For this PR, could you let `selectedTabPosition` in the `companion`? I'm not fan of the idea, but for now I just want to merge this PR.

Reply:
Yeah, I also not a big fan of such idea, but let's do for this PR.